test-case-reporting/index.ts

```diff
@@ -22,7 +22,7 @@ const app = express();
 const PORT = process.env.PORT || 8080;
 
 app.get('/', (req, res) => {
-  res.send('Hello, world, 2.0!');
+  res.send('Avast, ye deployed world!');
```

Comment:
Lets not clutter the PR and commit history with these changes :laughing:

Reply:
Unresolving until this change is reverted

Reply:
Sorry, I thought I reverted it. I'll get on that.

Reply:
Done.